src/platform/testing/e2e/test-server.js

```diff
@@ -30,9 +29,12 @@ if (options.unexpected && options.unexpected.length !== 0) {
 const app = express();
 
 const root = path.resolve(__dirname, `../../../../build/${options.buildtype}`);
+appSettings.parseFromBuildDir(root);
+const routes = appSettings.getAllApplicationRoutes();
+
 app.use(morgan('combined', { skip: (req, _res) => { return req.path.match(/(css|js|gif|jpg|png|svg)$/); } }));
 app.use(express.static(root));
-getRoutes(manifests).forEach(url => {
+routes.forEach(url => {
   app.use(url, fallback(`${url}/index.html`, { root }));
 });
```

Comment:
This is the server that serves files during E2E tests. I would honestly like to delete it and leverage Webpack as a static web server instead, but that wouldn't belong in this PR.